{"cve": {"data_type": "CVE", "data_format": "MITRE", "data_version": "4.0", "CVE_data_meta": {"ID": "CVE-2023-50559", "ASSIGNER": "cve@mitre.org"}, "problemtype": {"problemtype_data": [{"description": [{"lang": "en", "value": "NVD-CWE-Other"}]}]}, "references": {"reference_data": [{"url": "https://gist.github.com/nieeka/5edb552e7cf62cdb18decd9c4292d429", "name": "https://gist.github.com/nieeka/5edb552e7cf62cdb18decd9c4292d429", "refsource": "", "tags": ["Third Party Advisory"]}, {"url": "https://gist.github.com/nieeka/5edb552e7cf62cdb18decd9c4292d429", "name": "https://gist.github.com/nieeka/5edb552e7cf62cdb18decd9c4292d429", "refsource": "", "tags": ["Third Party Advisory"]}, {"url": "https://github.com/OpenXiangShan/XiangShan/issues/2534", "name": "https://github.com/OpenXiangShan/XiangShan/issues/2534", "refsource": "", "tags": ["Issue Tracking"]}, {"url": "https://github.com/OpenXiangShan/XiangShan/issues/2534", "name": "https://github.com/OpenXiangShan/XiangShan/issues/2534", "refsource": "", "tags": ["Issue Tracking"]}]}, "description": {"description_data": [{"lang": "en", "value": "An issue was discovered in XiangShan v2.1, allows local attackers to obtain sensitive information via the L1D cache."}]}}, "configurations": {"CVE_data_version": "4.0", "nodes": [{"operator": "OR", "children": [], "cpe_match": [{"vulnerable": true, "cpe23Uri": "cpe:2.3:a:openxiangshan:xiangshan:2.1:*:*:*:*:*:*:*", "cpe_name": []}]}]}, "impact": {"baseMetricV3": {"cvssV3": {"version": "3.1", "vectorString": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:N/A:N", "attackVector": "LOCAL", "attackComplexity": "LOW", "privilegesRequired": "LOW", "userInteraction": "NONE", "scope": "UNCHANGED", "confidentialityImpact": "HIGH", "integrityImpact": "NONE", "availabilityImpact": "NONE", "baseScore": 5.5, "baseSeverity": "MEDIUM"}, "exploitabilityScore": 1.8, "impactScore": 3.6}}, "publishedDate": "2023-12-30T00:15Z", "lastModifiedDate": "2024-11-21T08:37Z"}